scala-interpreter/build.sbt

```diff
@@ -18,3 +18,4 @@ import sbt.Tests.{Group, SubProcess}
  */
 
 libraryDependencies ++= Dependencies.sparkAll.value
+libraryDependencies += "com.github.jupyter" % "jvm-repr" % "0.1.0"
```

Comment:
Given that all of the interpreter output gets funneled through the JVM before being sent back, would it make more sense for us to use this library at the JVM level for all of them? Either way, seems like another issue to open for discussion.